lms/static/scripts/frontend_apps/components/ErrorDialogApp.js

```diff
@@ -35,41 +35,33 @@ export default function ErrorDialogApp() {
   }
 
   return (
-    <Modal
-      onCancel={() => null}
-      title={title}
-      withCloseButton={false}
-      withCancelButton={false}
-      contentClass="LMS-Dialog LMS-Dialog--medium"
-    >
-      <ErrorDisplay error={error} description={description}>
-        {error.errorCode === 'reused_consumer_key' && (
-          <>
-            <p>
-              This Hypothesis {"installation's"} consumer key appears to have
-              already been used on another site. This could be because:
-            </p>
-            <ul>
-              <li>
-                This consumer key has already been used on another site. A site
-                admin must{' '}
-                <Link target="_blank" href="https://web.hypothes.is/get-help/">
-                  request a new consumer key
-                </Link>{' '}
-                for this site and re-install Hypothesis.
-              </li>
-              <li>
-                This {"site's"} tool_consumer_instance_guid has changed. A site
-                admin must{' '}
-                <Link target="_blank" href="https://web.hypothes.is/get-help/">
-                  ask us to update the consumer key
-                </Link>
-                .
-              </li>
-            </ul>
-          </>
-        )}
-      </ErrorDisplay>
-    </Modal>
+    <ErrorModal description={description} error={error} title={title}>
```

Comment:
This now wraps an `ErrorModal` instead of a `Modal`, with no other meaningful changes.